add docs for central hostpath mapper (#1091)
loft-sh/vcluster: docs/pages/pro-features/central-hostpath-mapper.mdx

---
title: Centralized Hostpath Mapper
sidebar_label: Centralized Hostpath Mapper
---

This feature is an extention to the existing [hostpath mapper](https://www.vcluster.com/docs/operator/monitoring-logging#enabling-hostpath-mapper) feature of vcluster.
Currently when enabled, hostpath mapper can support the following usecases:
1. [Enabling container based logging used by tools like fluentd, logstash etc.](https://www.vcluster.com/docs/operator/monitoring-logging#logging-with-elk-and-fluentd-inside-vcluster) inside vcluster
2. [Enabling pod based logging used by loki](https://www.vcluster.com/docs/operator/monitoring-logging#logging-with-grafana-and-loki) inside vcluster
3. [Velero restic backups](https://www.vcluster.com/docs/operator/backup#using-velero-inside-vcluster) inside vcluster
4. Support for kubevirt workloads.

In the normal deployment of hostpath mapper which can be enabled by:
```yaml
hostpathMapper:
  enabled: true
```

A `daemonset` is spawned per node, per vcluster – which can be a resource hog as the number of `Nodes` and `vclusters` increase.

> To bypass this, the feature + component called Central Hostpath Mapper has been extended.

Reviewer: Potentially improving on clarity

```suggestion
The hostpath mapper feature was extended to a Central Hostpath Mapper to overcome this potential bottleneck.
```

> This is a loft integrated feature and the related docs can be found [here](https://loft.sh/docs/virtual-clusters/central-hostpath-mapper). Or you can follow the instructions below:

Reviewer: ```suggestion
This is a loft-integrated feature, and the related docs can be found [here](https://loft.sh/docs/virtual-clusters/central-hostpath-mapper).
Alternatively, one can follow the instructions below:
```
